design/use_cases/outdoor_localization_use_case.md

```diff
@@ -0,0 +1,15 @@
+# Outdoor Localization
+As a Robot user I want my robot to know its location on a given map of an outdoor area, such as a street or college campus, so that it can move around the area
```

Comment:
I would suggest that localization be handled external to the navigation stack -- packages like "robot_localization" already offer this outdoor localization and it is a whole system/project in it's own right. In that direction, I would also suggest that AMCL be moved external to navigation stack, so that when somebody finally make something from this millennium, we can start using it (without people thinking that AMCL is the "default" they should stick with)

Reply:
Agree that we should treat localization packages as something that can change, but still think the use case is relevant as we need to define the interfaces and message types being passed in order to make this possible. We may be the ones writing the "something from this millenium" in this case, I'm not assuming at all that we're using AMCL. That's a decision that we can discuss separately.

Reply:
I agree with Matt that even if it is an external package, we should consider/review its interface to allow for replacing it as a component.